7Cav-Alive.Altis/scripts/missionEnd/missionEnd_armor.sqf

```diff
@@ -0,0 +1,69 @@
+//Set a variable in the missionNameSpace to EndMissionNumber and set boxNumber equal to it.

```

Comment:
This code seems identical to the other code, why do we need more than one copy of it?

Reply:
Changed slightly.